queue_services/entity-filer/src/entity_filer/filing_processors/change_of_address.py

```diff
@@ -39,3 +41,21 @@ def process(business: Business, filing: Dict, filing_meta: FilingMeta):
             else:
                 address = create_address(new_address, k)
                 office.addresses.append(address)
+
+    if flag_on:
+        if business.in_dissolution:
+            batch_processings = BatchProcessing.find_by(business_id=business.id)
+            for batch_processing in batch_processings:
+                if batch_processing.status not in [
+                    BatchProcessing.BatchProcessingStatus.COMPLETED,
+                    BatchProcessing.BatchProcessingStatus.WITHDRAWN
+                ]:
+                    batch_processing.trigger_date = datetime.utcnow() + datedelta(days=62)
+                    batch_processing.meta_data = {
+                        **batch_processing.meta_data,
+                        'changeOfAddressDelay': True
+                    }
+                    target_dissolution_date = date.fromisoformat(batch_processing.meta_data['targetDissolutionDate'])
+                    target_dissolution_date += datedelta(days=62)
+                    batch_processing.meta_data['targetDissolutionDate'] = target_dissolution_date.isoformat()
+                    batch_processing.last_modified=datetime.utcnow()
```

Comment:
I think we need to account for the scenario where the trigger date will end up being more than 60 days.  

Basically, only the first COA for a business in dissolution will extend the trigger date.

![image](https://github.com/bcgov/lear/assets/6685223/b3711fa8-3483-4e05-9abb-9b3b5196b5ae)